resource_test.go

```diff
@@ -194,42 +227,143 @@ func TestGetResource(t *testing.T) {
 			Set("authorization", userToken).
```

Comment:
@tomipaul What is this test for ?

Reply:
@mikey2020 which of the tests? The one for invalid resource id or the one for nonexistent resource?